business/checkers/peer_authentication_checker.go

```diff
@@ -42,7 +42,7 @@ func (m PeerAuthenticationChecker) runChecks(peerAuthn security_v1beta.PeerAuthe
 		matchLabels = peerAuthn.Spec.Selector.MatchLabels
 	}
 	enabledCheckers = append(enabledCheckers, common.SelectorNoWorkloadFoundChecker(PeerAuthenticationCheckerType, matchLabels, m.WorkloadList))
-	if peerAuthn.Namespace == config.Get().IstioNamespace {
+	if peerAuthn.Namespace == config.Get().ExternalServices.Istio.RootNamespace {
```

Comment:
Didn't you wanted to use the new method you introduced in the previous PR? I guess it is a good moment to do so.

Same happens to a couple more places.

https://github.com/kiali/kiali/blob/add2f245d9739af879fd8d1fa73287730ef4209b/config/config.go#L871-L873

Reply:
so true... fixed!.